### PR TITLE
feat (User Management): add query params search and include_settings to list_users operation

### PR DESCRIPTION
**Pull request by PureMathematics**
## PR summary

Update Include_settings and substring search functionality (User Management) 

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information
the current issue for updating SDK https://github.ibm.com/IAM/uum/issues/5882 
the previous issue for the user management code change https://github.ibm.com/IAM/uum/issues/4948

### CODE REVIEW

File: usermanagementv1/user_management_v1_examples_test.go [hunk 1]
```diff
@@ -212,6 +211,57 @@ var _ = Describe(`UserManagementV1 Examples Tests`, func() {
 			fmt.Println(string(b))
 			// end-list_users
 		})
+		It(`ListUsers request search example`, func() {
+			fmt.Println("\nListUsers() result:")
+			// begin-list_users search
```
Comment (padamstx) [inline usermanagementv1/user_management_v1_examples_test.go:216]: Within the generated examples code, you cannot simply add new sections like this, because there can be at most one example code block for each operation.  The "// begin" and "// end" comments are markers for the begin and end of the code fragment to be used for the operation whose id is specified in those marker comments.
So there can only be one section with:
```
       // begin-list_users
       ...
       // end-list_users
```
This restriction is in place because we have a separate tool that can be used to extract these code fragments (one per operation) and compile them in a .json file that can be used when building the online API reference docs.

So, bottom line is that you can have only 1 example per operation and you need to leave the begin/end marker comments as they are.  If you want to incorporate the new parameters in the existing list_users example, feel free to do that.

**Review by padamstx — CHANGES_REQUESTED** [review]
Overall the PR looks good, except for the multiple example code fragments for the list_users operation.
Also, you are getting the failure in the DCO check because you did not sign-off your commit.  Next time you amend your commit, be sure to use the `-s` option.

File: usermanagementv1/user_management_v1_examples_test.go (hunk 1 above)
Comment (PureMathematics) [reply usermanagementv1/user_management_v1_examples_test.go]: I see, Thank you for the suggestion. Addressed.

**Review by padamstx — APPROVED** [review]
LGTM

**Comment by PureMathematics** [comment]
examples test 
Go-examples

**Comment by PureMathematics** [comment]
Integration test
Go-integration

**Review by padamstx — APPROVED** [review]
LGTM
